votingpool/example_test.go

```diff
@@ -17,109 +17,262 @@
 package votingpool_test
 
 import (
+	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
 	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcwallet/txstore"
 	"github.com/btcsuite/btcwallet/votingpool"
 	"github.com/btcsuite/btcwallet/waddrmgr"
 	"github.com/btcsuite/btcwallet/walletdb"
 	_ "github.com/btcsuite/btcwallet/walletdb/bdb"
 )
 
-func Example_basic() {
-	// This example demonstrates how to create a voting pool, create a
-	// series, get a deposit address from a series and lastly how to
-	// replace a series.
+var (
+	pubPassphrase  = []byte("pubPassphrase")
+	privPassphrase = []byte("privPassphrase")
+)
+
+func Example_depositAddress() {
+	// This example demonstrates how to create a voting pool with one
+	// series and get a deposit address for that series.
+
+	// Create the address manager and votingpool DB namespace.
+	mgr, vpNamespace, tearDownFunc, err := exampleCreateMgrAndDBNamespace()
+	if err != nil {
+		return
+	}
+	defer tearDownFunc()
+
+	// Create the voting pool.
+	pool, err := votingpool.Create(vpNamespace, mgr, []byte{0x00})
+	if err != nil {
+		fmt.Printf("Voting Pool creation failed: %v\n", err)
+		return
+	}
+
+	// Create a 2-of-3 series.
+	seriesID := uint32(0)
+	requiredSignatures := uint32(2)
+	pubKeys := []string{
+		"xpub661MyMwAqRbcFDDrR5jY7LqsRioFDwg3cLjc7tML3RRcfYyhXqqgCH5SqMSQdpQ1Xh8EtVwcfm8psD8zXKPcRaCVSY4GCqbb3aMEs27GitE",
+		"xpub661MyMwAqRbcGsxyD8hTmJFtpmwoZhy4NBBVxzvFU8tDXD2ME49A6JjQCYgbpSUpHGP1q4S2S1Pxv2EqTjwfERS5pc9Q2yeLkPFzSgRpjs9",
+		"xpub661MyMwAqRbcEbc4uYVXvQQpH9L3YuZLZ1gxCmj59yAhNy33vXxbXadmRpx5YZEupNSqWRrR7PqU6duS2FiVCGEiugBEa5zuEAjsyLJjKCh",
+	}
+	err = pool.CreateSeries(votingpool.CurrentVersion, seriesID, requiredSignatures, pubKeys)
+	if err != nil {
+		fmt.Printf("Cannot create series: %v\n", err)
+		return
+	}
+
+	// Create a deposit address.
+	addr, err := pool.DepositScriptAddress(seriesID, votingpool.Branch(0), votingpool.Index(1))
+	if err != nil {
+		fmt.Printf("Error generating DepositScriptAddress: %v\n", err)
+		return
+	}
+	fmt.Println("Generated deposit address:", addr.EncodeAddress())
+
+	// Output:
+	// Generated deposit address: 3QTzpc9d3tTbNLJLB7xwt87nWM38boAhAw
+}
+
+func Example_empowerSeries() {
+	// This example demonstrates how to empower a series by loading the private
+	// key for one of the series' public keys.
+
+	// Create the address manager and votingpool DB namespace.
+	mgr, vpNamespace, tearDownFunc, err := exampleCreateMgrAndDBNamespace()
+	if err != nil {
+		return
```

Comment:
I think you need to print the error here.  Otherwise any errors will result in no output, which is what the example output expects meaning it would pass the tests as if the example worked even though it didn't.